apps/site/components/withNavBar.tsx

```diff
@@ -10,19 +11,30 @@ import { useSiteNavigation } from '@/hooks';
 import { useRouter, usePathname } from '@/navigation.mjs';
 import { availableLocales } from '@/next.locales.mjs';
 
+import Button from './Common/Button';
+
 const WithNavBar: FC = () => {
   const { navigationItems } = useSiteNavigation();
   const { resolvedTheme, setTheme } = useTheme();
   const { replace } = useRouter();
   const pathname = usePathname();
 
+  const skipToContent = classNames(
+    'absolute left-0 top-0 m-3 -translate-y-16 bg-blue-500 p-3 transition-transform focus:translate-y-0 focus:outline-none'
+  );
+
   const locale = useLocale();
+  const t = useTranslations();
 
   const toggleCurrentTheme = () =>
     setTheme(resolvedTheme === 'dark' ? 'light' : 'dark');
 
   return (
     <div>
+      <Button className={skipToContent} href="#main">
```

Comment:
```suggestion
      <Button className="absolute left-3 top-3 -translate-y-16 focus:translate-y-0" href="#main">
```

Margin, padding, and background color don't seem to be needed 👀 Also when the button is focused, the outline must be visible, otherwise, it is not obvious that it is focused.

Reply:
Yep, not required! I added those previously just for testing, forgot to remove.